### DISCUSSION TITLE
Replacing FastAPI with Rust: Part 1 - Intro

### DISCUSSION THREAD

**Discussion by dbanty** [dbanty/dylananthony.com#9] [Posts]
Have questions or comments about [this blog post](https://dylananthony.com/posts/fastapi-rust-1-intro)? Leave them in this thread!

**Comment by anexa-digital** [comment]
Have you considered using FastAPI(Python) with Rust?   https://developers.redhat.com/blog/2017/11/16/speed-python-using-rust#python_with_itertools

**Comment by Adrian-Blandin** [comment]
Did you take a look at [poem](https://github.com/poem-web/poem)?
It has a package for openapi and a package to run in a Lambda (with examples).

**Reply by dbanty** [reply]
I did try it when it was very new and had a few sharp edges. I've been meaning to give it another go and write a blog post about it now that it has a few more versions under its belt. Have you tried it? How was your experience?

**Reply by Adrian-Blandin** [reply]
No, I did not try it yet, I was looking into it to use as a first Rust project (to learn, mostly), because I've read some comments on Reddit that recommended it.
